app/store/env/go.js

```diff
@@ -1166,6 +1166,29 @@ YUI.add('juju-env-go', function(Y) {
       userCallback(transformedData);
     },
 
+    setCharm: function(serviceName, charmUrl, force, callback) {
```

Comment:
Any comments here at all?  So we added a new api method to the Go API, that talks to the notification system which seems very intertwined. Is this pure spike code then?